tools/e2e-commons/pages/wp-admin/index.js

```diff
@@ -6,7 +6,6 @@ export { default as SimplePaymentBlock } from './blocks/simple-payments.js';
 export { default as WordAdsBlock } from './blocks/word-ads.js';
 
 export { default as DashboardPage } from './dashboard.js';
-export { default as InPlaceAuthorizeFrame } from './in-place-authorize.js';
```

Comment:
There are some bits left related to `async forceVariation` method: https://github.com/Automattic/jetpack/blob/master/tools/e2e-commons/pages/wp-admin/jetpack.js#L49